File: src/IsbInterop/Tasks/Wrappers/CustomWorkInfo.vb
Imports IsbInterop.Base
Imports IsbInterop.Base.Wrappers

Namespace Tasks.Wrappers

  ''' <summary>
  ''' Обертка над ICustomWorkInfo.
  ''' </summary>
  Friend Class CustomWorkInfo
    Inherits EdmsObjectInfo
    Implements ICustomWorkInfo

    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwICustomWorkInfo">COM-объект ICustomWorkInfo.</param>
    ''' <param name="scope">Область видимости.</param>
    Public Sub New(rcwICustomWorkInfo As Object, scope As IScope)
      MyBase.New(rcwICustomWorkInfo, scope)
    End Sub
  End Class
End NameSpace

File: src/IsbInterop.CSharp/Tasks/Wrappers/CustomWorkInfo.cs
using IsbInterop.Base;
using IsbInterop.Base.Wrappers;

namespace IsbInterop.Tasks.Wrappers
{
  /// <summary>
  /// Обертка над ICustomWorkInfo.
  /// </summary>
  internal class CustomWorkInfo : EdmsObjectInfo, ICustomWorkInfo
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwICustomWorkInfo">COM-объект ICustomWorkInfo.</param>
    /// <param name="scope">Область видимости.</param>
    public CustomWorkInfo(object rcwICustomWorkInfo, IScope scope)
      : base(rcwICustomWorkInfo, scope) { }
  }
}
